pypit/masterframe.py

```diff
@@ -37,44 +34,43 @@ class MasterFrame(object):
     """
     __metaclass__ = ABCMeta
 
-    def __init__(self, frametype, setup, settings):
-        # Start us up
+    def __init__(self, frametype, setup, spectrograph=None, directory_path=None, mode=None):
```

Comment:
I still think we should probably have a Master's ParSet,
but that can await a future PR.